ci/pipeline.yml

```diff
@@ -51,22 +51,27 @@ jobs:
       run:
         path: pipeline-tasks/ci/tasks/prepare-docker-build.sh
   - task: build
-    privileged: true
     config:
       platform: linux
       image_resource:
         type: registry-image
         source:
-          repository: vito/oci-build-task
+          repository: gcr.io/kaniko-project/executor
+          tag: debug
```

Comment:
RU sure `debug` is the right tag? Not `latest`?

Reply:
`debug` has `sh` but `latest` does not, so it makes hijacking useless.